src/libs/ajax/leonardo/models/disk-models.ts

```diff
@@ -61,33 +63,88 @@ export interface ListDiskItem {
 export type PersistentDisk = ListDiskItem | GetDiskItem;
 export type AppDataDisk = PersistentDisk;
 
-export interface PdType {
-  label: GoogleDiskType;
-  displayName: string;
+export interface GooglePdType {
+  value: GoogleDiskType;
+  label: string;
   regionToPricesName: string;
 }
 
+export interface AzurePdType {
+  value: AzureDiskType;
+  label: string;
+  // TODO: Pricing skuLetter: 'S'; Enable SSD types | 'E';
+}
+
+export type SharedPdType = GooglePdType | AzurePdType;
+export const AzurePersistentDiskOptions: AzurePdType[] = [
+  {
+    value: 'Standard_LRS',
+    label: 'Standard HDD',
+  },
+  // TODO: Disabled the SSD option and the test in
+  // AzurePersistentDiskInputTest test until SSD is properly implemented and tested.
+  // Main blocker: Cost calculation.
+  // {
+  //   value: 'StandardSSD_LRS',
+  //   label: 'Standard SSD',
+  // },
+];
+
 export type PDLabels = 'standard' | 'balanced' | 'ssd';
-export const pdTypes: Record<PDLabels, PdType> = {
+export const googlePdTypes: Record<PDLabels, GooglePdType> = {
   standard: {
-    label: 'pd-standard',
-    displayName: 'Standard',
+    value: 'pd-standard',
+    label: 'Standard',
     regionToPricesName: 'monthlyStandardDiskPrice',
   },
   balanced: {
-    label: 'pd-balanced',
-    displayName: 'Balanced',
+    value: 'pd-balanced',
+    label: 'Balanced',
     regionToPricesName: 'monthlyBalancedDiskPrice',
   },
   ssd: {
-    label: 'pd-ssd',
-    displayName: 'Solid state drive (SSD)',
+    value: 'pd-ssd',
+    label: 'Solid state drive (SSD)',
     regionToPricesName: 'monthlySSDDiskPrice',
   },
 };
 export type DecoratedPersistentDisk = {
-  diskType: PdType;
+  diskType: GooglePdType;
 } & Omit<PersistentDisk, 'diskType'>;
 
+export const GCPPersistentDiskOptions = [
+  { label: googlePdTypes.standard.label, value: googlePdTypes.standard },
```

Comment:
huh... what value is this object providing? It seems to just take a `PdType` and then pull out the label into another field. Why can't the call site just take a `pdType: googlePdTypes.x` and then access the label via `pdType.label`?